### DISCUSSION TITLE
Welcome to zipcodeR Discussions!

### DISCUSSION THREAD

**Discussion by gavinrozzi** [gavinrozzi/zipcodeR#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sacharose23** [comment]
Hi Gavin!
Great package! What year does the current data reflect? Also, will you be updating based on new census data?

**Comment by etolson71** [comment]
Thanks for putting this package out.  Can you add more decimals to the lat/longs in the package?  If you need a source, this one has many significant digits and is better populated than the one in the package (https://public.opendatasoft.com/explore/dataset/us-zip-code-latitude-and-longitude/table/).  Thanks again and great work!!!  Erik

**Comment by ghost** [comment]
Is distance calculated using haversine formula?
